firestore.rules

```diff
@@ -121,7 +126,8 @@ service cloud.firestore {
 
     match /pushTokens/{pushTokensId} {
       allow read: if isAdmin();
-      allow create, update: if documentFieldsCheckOut(["androidId", "createdAt", "deviceToken", "platform"], []) && pushTokenFieldsTypeValidate(request.resource.data);
+      allow create: if isAdmin() && documentFieldsCheckOut(["androidId","createdAt","deviceToken","platform"],[]) && pushTokenFieldsTypeValidate(request.resource.data);
```

Comment:
In english, we add a space after comma. But in programming, we forget this rule. Why?

Reply:
In fact, here you have removed the spaces after comma.